native/core/src/templates.rs

```diff
@@ -209,7 +209,7 @@ fn execute_value_template(
             Ok(TomlValue::Array(items))
         }
         TomlValue::Table(t) => {
-            let mut new_table = toml::map::Map::with_capacity(t.len());
+            let mut new_table = TomlMap::with_capacity(t.len());
```

Comment:
not sure how this is meant to be better...

Reply:
Fixed by https://github.com/acterglobal/a3/pull/837/commits/a7da96c2e967c2afd110ac001b0058bb9c82cbf1